### ISSUE TITLE
Sway crashes when external monitors are connected

### ISSUE THREAD

**Issue by JonnyMako** [swaywm/sway#4828]
This is the offending commit:
https://github.com/swaywm/wlroots/commit/515679e4fed1c7c762a8c269fca09cbb195a4323

```
[render/gles2/texture.c:250] Cannot create DMA-BUF texture: EGL extension unavailable
sway: backend/drm/renderer.c:274: copy_drm_surface_mgpu: Assertion `tex' failed.
```

[gdb.txt](https://github.com/swaywm/sway/files/3989267/gdb.txt)

**Comment by emersion** [comment]
Fixed by https://github.com/swaywm/wlroots/pull/1956

**Comment by JonnyMako** [comment]
> Fixed by [swaywm/wlroots#1956](https://github.com/swaywm/wlroots/pull/1956)

That fixes my issue. Thanks!
